react/features/analytics/handlers/AmplitudeHandler.js

```diff
@@ -0,0 +1,71 @@
+import amplitude from 'amplitude-js';
+
+import { getJitsiMeetGlobalNS } from '../../base/util';
+
+import AbstractHandler from './AbstractHandler';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Analytics handler for Amplitude.
+ */
+class AmplitudeHandler extends AbstractHandler {
+    /**
+     * Creates new instance of the Amplitude analytics handler.
+     *
+     * @param {Object} options -
+     * @param {string} options.amplitudeAPPKey - The Amplitude app key required
+     * by the Amplitude API.
+     */
+    constructor(options) {
+        super();
+
+        const { amplitudeAPPKey } = options;
+
+        if (!amplitudeAPPKey) {
+            logger.warn(
+                'Failed to initialize Amplitude handler, no tracking ID');
+
+            return;
+        }
+
+        this._enabled = true;
+
+        amplitude.getInstance().init(amplitudeAPPKey);
+    }
+
+    /**
+     * Sets the Amplitude user properties.
+     *
+     * @param {Object} props - The user portperties.
+     * @returns {void}
+     */
+    setUserProperties(props) {
+        if (this._enabled) {
+            amplitude.getInstance().setUserProperties(props);
+        }
+    }
+
+    /**
+     * Sends an event to Amplitude. The format of the event is described
+     * in AnalyticsAdapter in lib-jitsi-meet.
+     *
+     * @param {Object} event - The event in the format specified by
+     * lib-jitsi-meet.
+     * @returns {void}
+     */
+    sendEvent(event) {
```

Comment:
This method could return async `logEvent` chain to make it able to control the flow, and react to exception.